tox.ini

```diff
@@ -1,23 +1,20 @@
 [tox]
-envlist = black, flake8, pylint, py{26,27,36,37}, ensure_non_running_provider
+envlist =
```

Comment:
AFAIU this changes the default execution behavior. I would prefer to do the (fast) static analysis before running the possibly slower unit tests.

Reply:
Ok. So the preferred order should be
```ini
envlist = black, flake8, pylint, py{26,27,36,37,38}, molecule_{lint,syntax,test}, extra
```
?

Reply:
Yes, the order looks ok. I am a little bit hesitant of adding molecule to the default envlist since it requires docker/root privileges to run them IIRC. Therefore it makes more sense to run it explicitly.

Reply:
Good point, molecule is now outside of default `envlist` and can be run explicitly by `tox -e molecule`.